src/components/reusuables/content/ImageWithText.js

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { Row, Col } from "react-bootstrap"
+import { StaticImage } from "gatsby-plugin-image"
+import YohDevButton from "../YohDevButton"
+
+const ContentAll = (props) => {
+  const rowAlign = props.rowAlign === "image-right" ? "flex-md-row-reverse" : ""
+  return (
+    <div className="content-container">
+      <Row className={`justify-content-between ${rowAlign}`}>
+        <Col md={5} className="mb-sm-3">
+          <StaticImage
+            src={`../../../assets/images/cityBackground.jpg`}
+            className="img-fluid"
+          />
+        </Col>
+        <Col md={5}>
+          {props.contentHeading == "true" ? (
```

Comment:
This should be something like contentHeading && { }. "true" prop should be replaced with Boolean true.